source/reference/minio-server/minio-server.rst

```diff
@@ -2198,6 +2198,39 @@ endpoints as ``PRIMARY`` and ``SECONDARY`` respectively:
    :mc-conf:`notify_webhook comment <notify_webhook.comment>`
    configuration setting.
 
+.. _minio-server-envvar-object-lambda-webhook:
+
+Object Lambda
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following section documents environment variables for configuring MinIO to
+publish data to an HTTP webhook endpoint for an Object Lambda function. See
```

Comment:
"publish data" is probably not the most accurate description. Ideas? 

(Also these lines need to be joined into complete sentences.)

Reply:
"furnish data"? 
"provide data"?

Reply:
Fiddled with the text but kept `publish` after all. It seems to be a common way to describe sending data to a webhook endpoint.